### PR TITLE
Update UserAcceptableVerifiedCredentialCountsJob to use StatsD

### PR DESCRIPTION
**Pull request by rileyanderson**
## Summary
Run Sidekiq cron job daily to set StatsD gauges with `user_avc` totals. ` user_avc_totals.{proivider}.{scope}.total`

- use StatsD gauge for user_avc Metrics
- Updated [docs](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/AVC_IVC_logs.md)

## How to test
- Start server - `bundle exec rails c`
- In different terminal start sidekiq - `bundle exec sidekiq`
- In different terminal tail statsd logs - `tail -f log/statsd.log`
- Go to `http://localhost:3000/sidekiq/recurring-jobs`
- Enqueue - `UserAcceptableVerifiedCredentialTotals`
- Check logs for ` user_avc_totals.*.*.total` statsd keys

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#5325

## Testing done
- Specs
- Ran locally, StatsD metrics show up
- Checked logs for errors 
- checked authentication 

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated ([link to documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/AVC_IVC_logs.md))
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Comment by rileyanderson** [comment]
Closing for https://github.com/department-of-veterans-affairs/vets-api/pull/11854

**Comment by rileyanderson** [comment]
Reopening.  Want to replace current logging with StatsD

File: spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                expected_count,
+                :"#{expected_provider}_verified_account",
+                :"#{expected_scope}")
+  end
+
+  let(:expected_provider) { nil }
+  let(:expected_scope) { nil }
+  let(:expected_statsd_key) { "user_avc_totals.#{expected_provider}.#{expected_scope}.total" }
+  let(:expected_statsd_count) { expected_count }
+  let(:expected_count) { 5 }
```
Comment (bosawt) [inline spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb:51]: Just curious, would we be able to get this count from doing something like a `UserAVC.all.count`

File: spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      let(:expected_provider) { :mhv }
+
+      let!(:dslogon_avcs) do
+        create_list(:user_acceptable_verified_credential,
+                    expected_count,
+                    :dslogon_verified_account,
+                    :"#{expected_scope}")
+      end
+      let(:expected_statsd_key) { "user_avc_totals.mhv_dslogon.#{expected_scope}.total" }
+      let(:expected_statsd_count) { 10 }
```
Comment (bosawt) [inline spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb:106]: Same point here: Just curious, would we be able to get this count from doing something like a UserAVC.all.count

**Review by bosawt — APPROVED** [review]
I think this makes sense. My only comments would be around potential for performance issues, but I'm not super concerned since it's a Sidekiq job once a day. And if we ever see issues with the queries not completing or something we can revisit (I doubt this will be an issue)

File: spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb (hunk 1 above)
Comment (rileyanderson) [reply spec/jobs/identity/user_acceptable_verified_credential_totals_job_spec.rb]: Yeah that would make more sense

**Review by bosawt — APPROVED** [review]
Looks good!

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**